app/models/plum_schema.rb

```diff
@@ -1,328 +1,346 @@
 # frozen_string_literal: true
 module PlumSchema
   extend ActiveSupport::Concern
-  included do
-    # Hyrax
-    attribute :depositor
-    attribute :title
-    attribute :label
-    attribute :visibility
+  def imported_schema
```

Comment:
I like the idea of `imported_schema`, but it does make me think we should have a `MarcRelatorsSchema` etc. classes to clearly identify where some of these are coming from.